controllers/board-game-review-controllers.js

```diff
@@ -57,3 +57,16 @@ exports.newComment = (req, res, next) => {
         next(err)
     });
   };
+
+  exports.editReview = (req, res, next) => {
```

Comment:
in the past you have named these functions after the appropriate http method, I would keep this consistent and probs call this patchReview